### DISCUSSION TITLE
read access

### DISCUSSION THREAD

**Discussion by panditdhamdhere** [Cyfrin/foundry-full-course-cu#1826] [General]
even though giving read access in my foundry.toml file my test and script fails, anyone tell me what wrong i am doing
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/ff816165-ff16-4b47-bf05-095632455fe0)
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/4650b810-199d-400d-803a-5560b25d334f)

**Comment by heheboii11** [comment]
Hey can you provide the path id the happy svg you stored, If you can provide that it would be easy. Provide the screenshot of how the happy svg is saved or else, Ig your happy svg is not in img section, it is stored seperate as``` img.happy.svg```. if it is stored as said then give the permissions as ```{access = "read", path = "./img.happy.svg"}```, add this part in the ffi permissions.

**Reply by panditdhamdhere** [reply]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/ff23af51-b3e4-4c69-9534-fa444a5f339c)
Hi there you can see here

**Comment by urosognjenovic** [comment]
Try changing ```./img.happy.svg``` to ```./img/happy.svg```.

**Comment by EngrPips** [comment]
Hello @panditdhamdhere, @thebraveman17  solution should solve your issue I assume. Try it and let us know how it plays out

**Comment by heheboii11** [comment]
@thebraveman17 answer should solve your problem
